src/i18n/i18n.translations.ts

```diff
@@ -25,8 +26,8 @@ export const es = {
     [namespaces.pages.loginScreen]: loginScreenEs,
     [namespaces.pages.menuDeleteAccount]: menuDeleteAccountEs,
     [namespaces.pages.menuReferral]: menuReferralEs,
-    [namespaces.pages.notifications]: {title: "Notificaciones"},
-    [namespaces.pages.pickupPoint]: {title: "Punto de recogida", address: "Dirección"},
+    [namespaces.pages.notifications]: { title: "Notificaciones" },
+    [namespaces.pages.pickupPoint]: { title: "Punto de recogida", address: "Dirección" },
```

Comment:
Shall these two not follow a similar approach to the other ones? for instance:

`notificationsEs` and `pickingAddressEs`